packages/common/src/utils/challenges.ts

```diff
@@ -0,0 +1,147 @@
+import { ChallengeRewardID, OptimisticUserChallenge } from '../models'
```

Comment:
not gonna check this file too carefully, assuming it's the same as before but moved and git was dumb